README.md

```diff
@@ -42,7 +42,17 @@ Then start the contaier by running:
 $ docker start ovs-forwarder:$MLNX_OFED_VERSION
 ```
 Note: Make sure that you have vfs in switchdev mode and the vfs are binded before start the ovs-forwarder container  
-
+      In case of vf-lag (bonding with switchdev) the steps should be as following:  
+      - Create vfs on both mlnx ports  
```

Comment:
let write the exact command that they will need to run. It better to add this as section